lib/protocol/imeActivate.js

```diff
@@ -1,9 +1,9 @@
 /**
  *
- * Make an engines that is available (appears on the list returned by getAvailableEngines) active.
+ * Make a engine that is available (appears on the list returned by getAvailableEngines) active.
```

Comment:
I admit I am not an english native speaker but shouldn't it be an `an engine` here?